jobs/build/ocp4/build.groovy

```diff
@@ -240,6 +243,33 @@ def planBuilds() {
         currentBuild.description += "error during plan builds step:\n${err.getMessage()}\n"
         throw err
     }
+
+    // check whether the group should be built right now
+    buildPlan.freeze_automation = buildlib.doozer("${doozerOpts} config:read-group --default 'no' freeze_automation",
+                                          [capture: true]).trim()
+
+    wrap([$class: 'BuildUser']) {
+        buildPlan.builderEmail = env.BUILD_USER_EMAIL
+    }
+
+    echo "Group's freeze_automation flag: ${buildPlan.freeze_automation}"
+    echo "Builder email: ${buildPlan.builderEmail}"
+
+    if (buildPlan.freeze_automation == "yes") {
+        echo "All automation are currently disabled in group.yml."
+        buildPlan.is_permitted = false
+    }
+
+    if (buildPlan.freeze_automation == "manual" && buildPlan.builderEmail == null) {
```

Comment:
The `builderEmail` part is like a hack. It doesn't donate a `-1` but would be better to handle this situation better.

Reply:
Unfortunately, this is the most elegant way I know of given the current plugins. Happy to be proven wrong, though.